src/wp-content/themes/twentyseventeen/functions.php

```diff
@@ -439,8 +439,8 @@ function twentyseventeen_colors_css_wrap() {
 		$customize_preview_data_hue = 'data-hue="' . $hue . '"';
 	}
 	?>
-	<style type="text/css" id="custom-theme-colors" <?php echo $customize_preview_data_hue; ?>>
-		<?php echo twentyseventeen_custom_colors_css(); ?>
+	<style type="text/css" id="custom-theme-colors" <?php echo esc_attr( $customize_preview_data_hue ); ?>>
```

Comment:
```suggestion
	<style type="text/css" id="custom-theme-colors" <?php echo $customize_preview_data_hue; ?>>
```

The `$customize_preview_data_hue` variable should not be escaped:
`<style type="text/css" id="custom-theme-colors" data-hue=&quot;356&quot;>`
It is either an empty string or an attribute/value pair like `data-hue="356"`. The `absint()` function ensures that the hue value is always an integer.

---

I would also like to move the space before the `data-hue` attribute inside the `$customize_preview_data_hue` variable, and the `type` attribute is unnecessary, but those changes are not important.
```
	if ( is_customize_preview() ) {
		$customize_preview_data_hue = ' data-hue="' . $hue . '"';
	}
	?>
	<style id="custom-theme-colors"<?php echo $customize_preview_data_hue; ?>>
```